src/plopp/functions/plot.py

```diff
@@ -36,13 +38,21 @@ def plot(obj: Union[VariableLike, ndarray, Dict[str, Union[VariableLike, ndarray
         Aspect ratio for the axes.
     cbar:
         Show colorbar in 2d plots if ``True``.
+    coords:
+        If supplied, use these coords instead of the input's dimension coordinates.
+        The list can contain both strings and ``Variable``s.
+        In the case of a string, the coordinate with the corresponding name in the
+        input data array will be used. In the case of a ``Variable``, it will replace
+        the corresponding dimension coordinate directly.
```

Comment:
I think this sentence is mostly redundant, the "If supplied, use these coords instead of the input's dimension coordinates." above implies this. Talking too much about dimension-coords is just confusing, I think.